crates/e2e/tests/e2e/eth_flow.rs

```diff
@@ -124,6 +135,12 @@ async fn eth_flow_tx(web3: Web3) {
         receiver,
     };
 
+    let client = reqwest::Client::default();
+
+    crate::services::start_api(&contracts, &[]);
+    crate::services::start_autopilot(&contracts, &[]);
+    crate::services::wait_for_api_to_come_up().await;
```

Comment:
Since these are commonly used together (AFAICT, always?), should we have a:
```suggestion
    crate::services::start_protocol_and_wait(&contracts, &[]).await;
```

Reply:
Might do this in the future but don't want to do it in this PR because I'm not sure enough that this is right in every test yet.